eogrow/core/pipeline.py

```diff
@@ -156,6 +157,13 @@ def get_execution_arguments(self, workflow: EOWorkflow) -> List[Dict[EONode, Dic
             exec_args.append(single_exec_dict)
         return exec_args
 
+    def _prepare_execution(self) -> ExecutionKind:
```

Comment:
As you wrote, this could also be a property. But since it has a side-effect of initializing ray it might be a bit too misleading to have it as a property. Therefore, I'm ok if it remains a method.

Regarding the name alternatives, I was thinking about `init_execution` or even `init_processing` because this isn't limited to "EOExecution" as defined in eo-learn.

Reply:
Nice. Now that we switched to name `_init_processing`, should we also rename `ExecutionKind` into `ProcessingKind` or `ProcessingType`?